README.md

```diff
@@ -32,10 +32,10 @@ import matplotlib.pyplot as plt
 import caustics
 import torch
 
-cosmology = caustics.cosmology.FlatLambdaCDM()
-sie = caustics.lenses.SIE(cosmology=cosmology, name="lens")
-src = caustics.light.Sersic(name="source")
-lnslt = caustics.light.Sersic(name="lenslight")
+cosmology = caustics.FlatLambdaCDM()
+sie = caustics.SIE(cosmology=cosmology, name="lens")
+src = caustics.Sersic(name="source")
+lnslt = caustics.Sersic(name="lenslight")
```

Comment:
Indeed the signature is more convenient for advanced users. I expect beginners to get tripped up a few times, but it will be clear in the docs and also they will get very clear crashes if they do it wrong. I don't think this will create any silent errors, so it won't cause major problems even for beginners. Pyhton is not a LBYL language haha